ament_cmake_auto/cmake/ament_auto_package.cmake

```diff
@@ -87,6 +90,20 @@ macro(ament_auto_package)
     )
   endif()
 
+  # install directories to share
+  foreach(_dir ${_ARG_INSTALL_TO_SHARE})
+    if(IS_ABSOLUTE "${_dir}")
+      get_filename_component(_dir_name "${_dir}" NAME)
+    else()
+      set(_dir_name "${_dir}")
+    endif()
+
+    install(
+      DIRECTORY "${_dir}/"
+      DESTINATION "share/${PROJECT_NAME}/${_dir_name}"
```

Comment:
A simpler alternative would be to remove the `_dir_name` logic above and call:

```
install(
  DIRECTORY "${_dir}"
  DESTINATION "share/${PROJECT_NAME}"
```

Reply:
@dirk-thomas I tried this in the past and it result in the contents of `_dir` being installed into `share/${PROJECT_NAME}` rather than in creating `_dir` in `share/${PROJECT_NAME}`

Reply:
Not if your directory doesn't have a trailing slash.

Reply:
Turns out I had a trailing slash in the package where I was testing :)